core/templates/dev/head/library/Library.js

```diff
@@ -108,6 +108,29 @@ oppia.controller('Library', [
 
     var isAnyCarouselCurrentlyScrolling = false;
 
+    var highlightText = null;
+
+    $scope.highlight = function(index) {
+      var element = $('.oppia-library-group');
```

Comment:
Don't do this; it's not Angular-ish to refer to stuff in the DOM and manipulate it. Instead, you can just have a $scope.currentlyHighlightedGroupIndex that is generally null; this method sets it to some index and the other method sets it back to null. Then, in the HTML, you can use ng-if on whether currentlyHighlightedGroupIndex === $index to add the ">".